GLTFSDK/Inc/GLTFSDK/Schema.h

```diff
@@ -3,19 +3,93 @@
 
 #pragma once
 
-#include "SchemaJson.h" // autogenerated header
-
 #include <string>
+#include <memory>
 #include <unordered_map>
 
 namespace Microsoft
 {
     namespace glTF
     {
-        namespace GLTFSchema
+        constexpr const char SCHEMA_URI_GLTF[] = "glTF.schema.json";
```

Comment:
Add a bug to have these added in SchemaJson?

Reply:
https://github.com/Microsoft/glTF-SDK/issues/26